src/System.Data.OleDb/src/Resources/Strings.resx

```diff
@@ -0,0 +1,1020 @@
+    <value>Information used to connect to a Data Source.</value>
+  </data>
+  <data name="OleDbConnection_ConnectionTimeout" xml:space="preserve">
+    <value>Current connection timeout value, 'Connect Timeout=X' in the ConnectionString.</value>
+  </data>
+  <data name="OleDbConnection_Provider" xml:space="preserve">
+    <value>Current OLE DB provider ProgID, 'Provider=X' in the connection string.</value>
+  </data>
+  <data name="OleDbConnection_ServerVersion" xml:space="preserve">
+    <value>Version of the product accessed by the OLE DB Provider.</value>
+  </data>
+  <data name="OleDbCommandBuilder_DataAdapter" xml:space="preserve">
+    <value>The DataAdapter for which to automatically generate OleDbCommands</value>
+  </data>
+  <data name="OleDbParameter_OleDbType" xml:space="preserve">
+    <value>The parameter native type.</value>
+  </data>
+  <data name="PlatformNotSupported_OleDb" xml:space="preserve">
+    <value>System.Data.OleDb is not supported on this platform.</value>
+  </data>
+  <data name="PlatformNotSupported_GetIDispatchForObject" xml:space="preserve">
+    <value>Marshal.GetIDispatchForObject API not available on this plaTform</value>
+  </data>
+  <data name="ADP_CalledTwice" xml:space="preserve">
+    <value>The method '{0}' cannot be called more than once for the same execution.</value>
+  </data>
+  <data name="ADP_IncorrectAsyncResult" xml:space="preserve">
+    <value>Incorrect async result.</value>
+  </data>
+  <data name="ADP_SingleValuedProperty" xml:space="preserve">
+    <value>The only acceptable value for the property '{0}' is '{1}'.</value>
+  </data>
+  <data name="ADP_DoubleValuedProperty" xml:space="preserve">
+    <value>The acceptable values for the property '{0}' are '{1}' or '{2}'.</value>
+  </data>
+  <data name="ADP_InvalidPrefixSuffix" xml:space="preserve">
+    <value>Specified QuotePrefix and QuoteSuffix values do not match.</value>
+  </data>
+  <data name="ADP_BadParameterName" xml:space="preserve">
+    <value>Specified parameter name '{0}' is not valid.</value>
+  </data>
+  <data name="ADP_MultipleReturnValue" xml:space="preserve">
+    <value>Multiple return value parameters are not supported.</value>
+  </data>
+  <data name="ADP_EmptyString" xml:space="preserve">
+    <value>Expecting non-empty string for '{0}' parameter.</value>
+  </data>
+  <data name="ADP_EmptyArray" xml:space="preserve">
+    <value>Expecting non-empty array for '{0}' parameter.</value>
+  </data>
+  <data name="ADP_InvalidArgumentLength" xml:space="preserve">
+    <value>The length of argument '{0}' exceeds it's limit of '{1}'.</value>
+  </data>
+  <data name="ADP_MustBeReadOnly" xml:space="preserve">
+    <value>{0} must be marked as read only.</value>
+  </data>
+  <data name="ConfigProviderNotFound" xml:space="preserve">
+    <value>Unable to find the requested .Net Framework Data Provider.  It may not be installed.</value>
+  </data>
+  <data name="ConfigProviderInvalid" xml:space="preserve">
```

Comment:
Several more .NET Framework mentions to replace. Probably just change ".Net Framework Data Provider" -> "data provider" ?